automation/patch_igz/patch_remote.py

```diff
@@ -590,13 +621,42 @@ def _resolve_overwrite_registry(self):
     "-lc",
     "--log-collector",
     is_flag=True,
-    help="Deploy the log collector as well",
+    help="Deploy the log collector",
+)
+@click.option(
+    "-ml",
+    "--mlrun",
+    is_flag=True,
+    help="Deploy the mlrun image",
+)
+@click.option(
+    "-ap",
+    "--api",
+    is_flag=True,
+    help="Deploy the mlrun API image",
 )
```

Comment:
Thinking out loud: Maybe we should make this "True" by default, and change it to "skip-api", as most of the time we will use this script to patch the api